swxsoc/swxdata.py

```diff
@@ -328,9 +328,9 @@ def global_attribute_template(
                     f"Instrument, {instr_name}, is not recognized. Must be one of {swxsoc.config['mission']['inst_names']}."
                 )
             # Set the Property
-            meta["Descriptor"] = (
-                f"{instr_name.upper()}>{swxsoc.config['mission']['inst_to_fullname'][instr_name]}"
-            )
+            meta[
+                "Descriptor"
+            ] = f"{instr_name.upper()}>{swxsoc.config['mission']['inst_to_fullname'][instr_name]}"
```

Comment:
Was this black formatting that did this?

Reply:
yes ^